src/tests/multihost/alltests/conftest.py

```diff
@@ -559,6 +559,62 @@ def del_sudo_rule():
 # ====================  Class Scoped Fixtures ================
 
 
+@pytest.fixture(scope='class')
+def create_custom_posix_usersgroups(session_multihost, setup_sssd_krb, create_posix_usersgroups, krb_connection_timeout):
```

Comment:
The fixture needs a finalizer to remove the created users and groups properly, otherwise it will crash on second run.
(Which will happen when the tests are split in tie1 and tier2).